source/GameAction.cpp

```diff
@@ -214,6 +215,16 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "relocate")
+	{
+		for(const DataNode &grand : child)
+		{
+			if(grand.Token(0) == "planet" && grand.Size() > 1)
+				teleportPlanet = GameData::Planets().Get(grand.Token(1));
```

Comment:
I see a comment about changing the vocabulary from "teleport" to "relocate" everywhere. You said "done," but this still says "teleportPlanet."

Reply:
done?